src/taipy/core/common/_utils.py

```diff
@@ -52,4 +52,4 @@ def _fcts_to_dict(objs):
     return [d for obj in objs if (d := _fct_to_dict(obj)) is not None]
 
 
-Subscriber = namedtuple("Subscriber", "callback params")
+_Subscriber = namedtuple("Subscriber", "callback params")
```

Comment:
I think mypy once complained with me that the namedtuple and the variable name don't line up so I think this would be better

```suggestion
_Subscriber = namedtuple("_Subscriber", "callback params")
```